tests/end2end/features/urlmarks.feature

```diff
@@ -185,4 +196,18 @@ Feature: quickmarks and bookmarks
     Scenario: Deleting a quickmark
         When I run :quickmark-add http://localhost:(port)/data/numbers/15.txt fifteen
         And I run :quickmark-del fifteen
-        Then the quickmark file should not contain "fourteen http://localhost:*/data/numbers/15.txt "
+        Then the quickmark file should not contain "fifteen http://localhost:*/data/numbers/15.txt "
+
+    Scenario: Deleting the current page's quickmark if it has none
+        When I open about:blank
+        And I run :quickmark-del
+        Then the error "Quickmark for 'about:blank' not found!" should be shown
+
+    Scenario: Deleting the current page's quickmark
+        When I open data/numbers/7.txt
+        And I run :quickmark-save
+        And I wait for "Entering mode KeyMode.prompt (reason: question asked)" in the log
+        And I press the keys "seven"
+        And I press the keys "<Enter>"
```

Comment:
You can use `When I run :quickmark-add http://localhost:(port)/data/numbers/7.txt seven` instead.